dashboard/templates/dashboard/includes/user_profile_post_comments.html

```diff
@@ -0,0 +1,48 @@
+{% load staticfiles %}
+<div class="sd-profile-comments-title">
+    <h5>Comments</h5>
+</div>
+<div id="sd-profile-comments-wrapper">
+        {% if user_posts %}
+            <div class="media-list">
+                {% for post in user_posts %}
```

Comment:
Changing this to {% for comment in post.comments.all %} and switching anything with post to comment in the below lines should fix it.